CatProducer/plugins/RecoEventInfoProducer.cc

```diff
@@ -120,11 +120,14 @@ void RecoEventInfoProducer::produce(edm::Event& event, const edm::EventSetup& ev
       const unsigned int trigIndex = hltConfig_.triggerIndex(trigName);
       if ( trigIndex < hltHandle->size() )
       {
-        if ( hltHandle->accept(trigIndex) ) { isPassed = true; break; }
+	if ( hltHandle->accept(trigIndex) ) {
+	  const std::pair<int,int> prescales = hltConfig_.prescaleValues(event, eventSetup, trigName);
+	  psValue = prescales.first * prescales.second;
+	  break;
```

Comment:
Is the break; statement really needed? I think it can introduce some side effect when combined with multi-paths.
If two paths Path1(PS=10) and Path2(PS=1) are fired, it could be better to return results based on the Path2 but it will return PS factor 10.

The config in this PR splits all paths, so it is safe at this moment.

BTW, is it right to multiply prescale.first and .second? I guess one of them are just PS column number (maybe wrong)